src/screens/LoginScreen/style.ts

```diff
@@ -1,13 +1,43 @@
 import { StyleSheet, ViewStyle } from 'react-native';
+import { COLORS } from '../../COLORS';
 
-type LoginScreenStylesType = {
+export type LoginScreenStylesType = {
   container: ViewStyle;
+  googleButton: ViewStyle;
+  facebookButton: ViewStyle;
+  fontFacebook: ViewStyle;
+  fontGoogle: ViewStyle;
 };
 export const style = StyleSheet.create<LoginScreenStylesType>({
   container: {
-    padding: 10,
-    flexDirection: 'column',
-    justifyContent: 'space-between',
     flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  googleButton: {
+    width: '100%',
+    backgroundColor: COLORS.dawnPink,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
```

Comment:
your style for google and facebook differs only in color 

I see two solutions
1) put this style in your button component as default and add your additional style from props 
2) in this style file create `const `with common styles and put for `googleButton` and 'facebookButton' using destructuring

the same for fonts style